README.md

```diff
@@ -99,12 +99,23 @@ The following example shows how to create a transaction message:
 ```python
 alice_keys = KeyPair.generate()
 
-cryptocurrency_service_name = 'exonum-cryptocurrency-advanced:0.12.0'
-loader.load_service_proto_files(runtime_id=0, service_name=cryptocurrency_service_name)
-
-cryptocurrency_module = ModuleManager.import_service_module(cryptocurrency_service_name, 'service')
-
-cryptocurrency_message_generator = MessageGenerator(instance_id=1024, artifact_name=cryptocurrency_service_name)
+cryptocurrency_artifact_name = "exonum-cryptocurrency-advanced"
+cryptocurrency_artifact_version = "0.12.0"
```

Comment:
Nit: "0.12.0" / "0.13.0-rc.2" inconsistency within docs.

Reply:
Yes. I think it makes sense to update it everywhere before release.